### DISCUSSION TITLE
[question] Using redis/limiter plugin on docker instead of filtron/morty

### DISCUSSION THREAD

**Discussion by Kopernikus1979** [searxng/searxng#1054] [Q&A]
Hi,

Do I need to also install Redis or is this already included in the searxng docker?
So what I did:

Set limiter & image_proxy & static_use_hash to "true"
Set redis url to: "url: redis://redis:6379/0"

All seems te be working since I got no errors but I want to be sure before making it public.

Thx

**Comment by dalf** [comment]
You can look at the docker logs to check for errors.
You can make sure the limiter plugin is enabled by looking at https://.../config (the plugins section)

**Comment by Kopernikus1979** [comment]
Hi,

All running fine now with the official redis docker container.
They also have an Alpine based version would this suit better ?

Are their any plans to implement redis into to the searxng docker container?
Now my searxng docker connects trough redis://ipofredisdocker?password=mypassword
All seems to be working fine
